server/src/main/java/org/opensearch/index/IndexModule.java

```diff
@@ -166,6 +169,18 @@ public final class IndexModule {
         Property.NodeScope
     );
 
+    /** Which lucene file extensions to load with nio. All others will default to mmap.
+     *  This is an expert setting.
+     *  @see <a href="https://lucene.apache.org/core/9_2_0/core/org/apache/lucene/codecs/lucene92/package-summary.html#file-names">Lucene File Extensions</a>.
```

Comment:
Although the file extension list looks same, you should ideally be using latest codec which is 9.5 right now - https://lucene.apache.org/core/9_5_0/core/org/apache/lucene/codecs/lucene95/package-summary.html#file-names

Reply:
Good catch